docs/guide-send-tx-bundle.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Guide to Sending Transactions and Bundles on Flashbots
+---
+
+Navigating the various methods for submitting transactions and bundles on Flashbots can be overwhelming. This guide aims to clarify the available options, helping you choose the most suitable one.
```

Comment:
The first sentence here is a bit negative. Prefer more neutral / constructive phrasing -- eg. "There are several ways to submit transactions and bundles to Flashbots. This guide will help you choose the most suitable option for your use case."